testcontainers/src/watchdog.rs

```diff
@@ -8,22 +8,29 @@ use std::{collections::BTreeSet, sync::Mutex, thread};
 
 static WATCHDOG: Lazy<Mutex<Watchdog>> = Lazy::new(|| {
     thread::spawn(move || {
-        let signal_docker = Cli::default();
-        let mut signals =
-            Signals::new([SIGTERM, SIGINT, SIGQUIT]).expect("failed to register signal handler");
+        let rt = tokio::runtime::Builder::new_current_thread()
```

Comment:
I personally like to avoid using abbreviations (except for widely known ones like e.g. `HTTP`) for more clarity inside the code.